via-env

```diff
@@ -1,32 +1,38 @@
+#!/bin/sh
```

Comment:
The shebang is mostly an indicator of what the script is.  Here I'm saying "POSIX shell".

Reply:
This is not a big deal but just for clarity. #!/bin/sh is only useful if calling the script directly. But since the script exports variables it's not designed to be called that way. Since this would create a sub process and the parent shell would not see those variables.

So #!/bin/sh is not a real indicator of what shell is is being used. I used bash isms for that reason, and I make an assumption the bash is the current user shell.

This is kinda a digression, the main thing is this source script works for you. My intention is not to distribute this to other users. It was meant sanitize the via environment for you.